crates/driver/src/infra/config/file/mod.rs

```diff
@@ -179,10 +187,22 @@ struct SolverConfig {
     /// The account which should be used to sign settlements for this solver.
     account: Account,
 
-    /// Maximum time allocated to wait for a solver response to propagate to the
-    /// driver.
+    /// Maximum time allocated for http request/reponse to propagate through
+    /// network.
     #[serde(default = "default_http_time_buffer_milliseconds")]
-    http_time_buffer_miliseconds: u64,
+    http_time_buffer_milliseconds: u64,
```

Comment:
Can we group these into a single Timeout type to make the config file more logically coherent and readable?

Reply:
Fixed.